lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -1632,7 +1632,9 @@ struct FIRRTLLowering : public FIRRTLVisitor<FIRRTLLowering, LogicalResult> {
   LogicalResult visitExpr(LTLUntilIntrinsicOp op);
   LogicalResult visitExpr(LTLEventuallyIntrinsicOp op);
   LogicalResult visitExpr(LTLClockIntrinsicOp op);
-  LogicalResult visitExpr(LTLDisableIntrinsicOp op);
+
+  template <typename TargetOp, typename IntrinsicOp>
```

Comment:
minor: Maybe give this a different name?  giving it a similar name to the visitor-derived functions is slightly misleading.